### PR TITLE
Update et_ee.lang

### PR DESCRIPTION
**Pull request by Madis0**

### CODE REVIEW

**Comment by thexaero** [comment]
Thank you very much! I will try to review it soon.

File: et_ee.lang [hunk 1]
```diff
@@ -415,3 +416,55 @@ gui.xaero_box_pushbox_potion_effects=Läbipaistev kast ümber vanilli võlujoogi
...
+gui.xaero_minimap_frame=Raami stiil
+gui.xaero_minimap_frame_color=Raami värv
+gui.xaero_box_minimap_frame_color=Minikaardi raami värv mitte-vaikimisi värvitud stiilide korral.
+gui.xaero_minimap_frame_default=vaikimisi
+gui.xaero_minimap_frame_colored_thick=värvitud
+gui.xaero_minimap_frame_colored_thin=värvitud õhuke
+gui.xaero_compass_north=N
+gui.xaero_compass_east=E
+gui.xaero_compass_south=S
+gui.xaero_compass_west=W
```
Comment (thexaero) [inline et_ee.lang:428]: Are these untranslated on purpose?

**Review by thexaero — COMMENTED** [review]
Thank you very much! I only have 1 question for which I left a comment for.

File: et_ee.lang (hunk 1 above)
Comment (Madis0) [reply et_ee.lang:428]: Yes, Estonian uses full words of 3-5 characters so I thought it looks better with English here, like a compass.

Comment (thexaero) [reply et_ee.lang:428]: Alright, great!

**Pull request merged**